docs/index.asciidoc

```diff
@@ -463,6 +575,55 @@ option when authenticating to the Elasticsearch server. If set to an
 empty string authentication will be disabled.
 
 
+[id="plugins-{type}s-{plugin}-deprecated-options"]
+==== Elasticsearch Input deprecated configuration options
+
```

Comment:
This is a departure from the standard approach we've used for deprecated options in other plugins. For example, https://www.elastic.co/guide/en/logstash/current/plugins-inputs-beats.html#plugins-inputs-beats-options.  

This approach is cleaner, but not as discoverable for users with older configs.